### PR TITLE
GLT-248 fix List X pages

### PR DESCRIPTION
**Pull request by callunity**
* add location column to Sample, Library and Pool
* fix backwards ordering of pagination buttons

### CODE REVIEW

**Comment by apmasell** [comment]
:+1:

File: miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java [hunk 1]
```diff
@@ -1049,7 +1054,7 @@ public static BoxableDto asDto(Boxable from, String position) {
     dto.setId(from.getId());
     dto.setAlias(from.getAlias());
     dto.setBoxAlias(from.getBoxAlias());
-    dto.setBoxPosition(from.getBoxPosition());
+    dto.setBoxPosition(BoxUtils.makeLocationLabel(from));
```
Comment (djcooke) [inline miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java]: The overwrites an above setting which uses the `String position` parameter

File: miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java [hunk 2]
```diff
@@ -1117,6 +1121,10 @@ public static PoolDto asDto(Pool<? extends Poolable<?, ?>> from) {
       }
     }
     dto.setPooledElements(pooledElements);
+    if (!isStringEmptyOrNull(from.getIdentificationBarcode())) {
+      dto.setIdentificationBarcode(from.getIdentificationBarcode());
```
Comment (djcooke) [inline miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java]: already set above, though without the null/empty check

File: miso-web/src/main/webapp/pages/editSample.jsp [hunk 3]
```diff
@@ -868,6 +868,8 @@
       </ul>
   --%>
   <a name="library"></a>
+  
+  <c:if test="${ detailedSample and sampleCategory eq 'Aliquot' }">
```
Comment (djcooke) [inline miso-web/src/main/webapp/pages/editSample.jsp]: This should be shown for plain samples as well
(`!detailedSample or sampleCategory eq 'Aliquot'`)

**Comment by djcooke** [comment]
:+1: after comments are addressed

**Pull request merged**